lib/src/main/java/io/ably/lib/push/Storage.java

```diff
@@ -0,0 +1,45 @@
+package io.ably.lib.push;
+
+import java.lang.reflect.Field;
+
+/**
+ * Interface for an entity that supplies key value store
+ */
+public interface Storage {
+
+    /**
+     * Insert string value in to storage
```

Comment:
`Insert` is the wrong verb to use in this context, as if there's an existing value then it'll be replaced.

Reply:
https://github.com/ably/ably-java/pull/681/commits/16bf5233f352b5817f5afae1af1e95ab7e283c7f